### PR TITLE
docs(2021-01-06): add minutes for fe meeting

### PR DESCRIPTION
**Pull request by paschalidi**
discussion https://github.com/dhis2/notes/discussions/192

### CODE REVIEW

**Review by HendrikThePendric — APPROVED** [review]
The parts of the meeting I was actively involved in all have been summarised well.

File: decisions/2021/01/14-frontend-initiatives.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        Discussion: Austin gave progress update.
+
+    - Continuous Delivery & App Hub updates - [@Birkbjo](https://github.com/Birkbjo) [@amcgee](https://github.com/amcgee)
+
+      Discussion: Birk gave update on the topic. There is wip on the topic. 
+
+      Two things we want to look into is a) we want to be able to enable continuous delivery. b) We want to have the authentication with the backend in place.
+
+    - Extra topic 
+      Discussion: Hendrik and Austin will connect with Ulaf regarding the data quality app.
```
Comment (cooper-joe) [inline decisions/2021/01/14-frontend-initiatives.md:86]: Olav, rather than Ulaf. (I think!)

File: decisions/2021/01/14-frontend-initiatives.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+    - Responsive Dashboard
+    
+        Discussion: Jan is making progress on the work. Milagros and Jeniffer are the main people working on it.
+        Jeniffer works on the responsivness of the app. This part is in PR. Overall is looking pretty good. 
+        Milagros, is work in progress. Couple of details is left.
+
+    - Offline design update from Joe
+    
+        Discussion: Joe has a design proposal for offline connectivity for mobile and desktop. The concept is there and it is basically ready to be implemented. Mobule version is 90 percent there.
```
Comment (cooper-joe) [inline decisions/2021/01/14-frontend-initiatives.md:107]: We can add this link for the offline design proposal: https://jira.dhis2.org/browse/LIBS-48

File: decisions/2021/01/14-frontend-initiatives.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+        There will be an upcoming meeting with Jan, Joe
+
+  - UI & Platform updates - [@ismay](https://github.com/ismay) [@HendrikThePendric](https://github.com/HendrikThePendric) [@cooper-joe](https://github.com/cooper-joe) [@amcgee](https://github.com/amcgee)
+
+    * ui cons 
+
+      Discussion: are released and people can use it. There is also a strategy for migration that Joe has worked on. There is a breaking change to ui v5, the change is that if you have been using the old icons you want to be careful because the old icons have been removed.
```
Comment (cooper-joe) [inline decisions/2021/01/14-frontend-initiatives.md:114]: The ui-icons migration strategy is available for dashboards, data visualizer and (soon) maps. Other apps do not have a migration strategy available yet.

File: decisions/2021/01/14-frontend-initiatives.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+        Milagros, is work in progress. Couple of details is left.
+
+    - Offline design update from Joe
+    
+        Discussion: Joe has a design proposal for offline connectivity for mobile and desktop. The concept is there and it is basically ready to be implemented. Mobule version is 90 percent there.
+        There will be an upcoming meeting with Jan, Joe
+
+  - UI & Platform updates - [@ismay](https://github.com/ismay) [@HendrikThePendric](https://github.com/HendrikThePendric) [@cooper-joe](https://github.com/cooper-joe) [@amcgee](https://github.com/amcgee)
+
+    * ui cons
```
Comment (cooper-joe) [inline decisions/2021/01/14-frontend-initiatives.md:112]: ui-icons

**Review by cooper-joe — APPROVED** [review]
Thanks for taking notes 👍 I've noted a couple of additions/typos.

File: decisions/2021/01/14-frontend-initiatives.md (hunk 1 above)
Comment (paschalidi) [reply decisions/2021/01/14-frontend-initiatives.md:86]: Good point right!

File: decisions/2021/01/14-frontend-initiatives.md (hunk 4 above)
Comment (paschalidi) [reply decisions/2021/01/14-frontend-initiatives.md:112]: cool cool thanks Joe

**Review by ghost — APPROVED** [review]

File: decisions/2021/01/14-frontend-initiatives.md (hunk 1 above)
Comment (JoakimSM) [inline decisions/2021/01/14-frontend-initiatives.md]:
```suggestion
      Discussion: Hendrik and Austin will connect with Olav regarding the data quality app.
```

Comment (paschalidi) [reply decisions/2021/01/14-frontend-initiatives.md]: Updated now :)

**Comment by janhenrikoverland** [comment]
Looks good, @paschalidi. Just pushed a few minor changes.

**Pull request merged**